rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section5/rule_5_31.json

```diff
@@ -82,7 +82,7 @@
                                 ]
                             }
                         ],
-                        "type": "USER"
+                        "type": "BASELINE_0"
```

Comment:
@JasonGlazer @JacksonJ-KC Could you one of you reflect this change in the section 5 spreadsheet? I believe when section 5 was renumbered, this part was changed unexpectedly somehow.

Reply:
This test json for rule 5-31 does not align with the section 5 spreadsheet - the `test_description`s belong to a different rule. The spreadsheet appears to be correct.